doc/getting_started.rst

```diff
@@ -0,0 +1,185 @@
+
+Both ``X`` and ``y`` are usually expected to be numpy arrays, though other
+formats are also sometimes accepted (e.g. sparse matrices and the more
+general :term:`array-like`).
+
+Once the estimator is fitted, it can be used for predicting target values of
+new data. You don't need to re-train the estimator::
+
+  >>> clf.predict(X)  # predict classes of the training data
+  array([0, 1])
+  >>> clf.predict([[4, 5, 6], [14, 15, 16]])  # predict classes of new data
+  array([0, 1])
+
+Transformers and Pipelines
+--------------------------
+
+Machine learning models are often composed of different parts. A typical
+model consists of a pre-processing step that transforms the data, and a
+final predictor that predicts target values.
+
+In scikit-learn, pre-processors and transformers follow the same API as the
+estimators objects (they actually all inherit from the same
+``BaseEstimator`` class). These objects don't have a :term:`predict` method
+but rather a :term:`transform` method that outputs a newly transformed
+sample matrix ``X``::
+
+  >>> from sklearn.preprocessing import StandardScaler
+  >>> X = [[0, 15],
+  ...      [1, -10]]
+  >>> StandardScaler().fit(X).transform(X)
+  array([[-1.,  1.],
+         [ 1., -1.]])
+
+Transformers and estimators (predictors) can be combined together into a single
+unifying object: a :class:`~sklearn.pipeline.Pipeline`.
+
+In the following example, we :ref:`load the Iris dataset <datasets>`, split it
+into train and test sets, and compute its accuracy score on the test data. The
+pipeline offers the same API as a regular estimator: it can be fitted and
+used for predictions with ``fit`` and ``predict``::
+
+  >>> from sklearn.preprocessing import StandardScaler
+  >>> from sklearn.linear_model import LogisticRegression
+  >>> from sklearn.pipeline import make_pipeline
+  >>> from sklearn.datasets import load_iris
+  >>> from sklearn.model_selection import train_test_split
+  >>> from sklearn.metrics import accuracy_score
+  ...
+  >>> preprocessor = StandardScaler()
+  >>> predictor = LogisticRegression(random_state=0)
+  >>> pipe = make_pipeline(preprocessor, predictor)
+  ...
+  >>> X, y = load_iris(return_X_y=True)
+  >>> X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=0)
+  >>> pipe.fit(X_train, y_train)  # fit the preprocessor first, then the predictor
+  Pipeline(steps=[('standardscaler', StandardScaler()),
+                  ('logisticregression', LogisticRegression(random_state=0))])
+
+  >>> accuracy_score(pipe.predict(X_test), y_test)
+  0.97...
+
+Model evaluation
+----------------
+
+Once a model has been fitted, it is natural to evaluate its performance on
+unseen data. We have just seen the
+:func:`~sklearn.model_selection.train_test_split` helper, but scikit-learn
+provides many other tools for model evaluation, in particular for
+:ref:`cross-validation <cross_validation>`.
+
+We here briefly show how to perform a 3-folds cross-validation using the
```

Comment:
we have moved away from a default 3 fold to a default 5 fold (IIRC), which I think should be what we present here.